scico/test/linop/xray/test_xray.py

```diff
@@ -49,7 +49,7 @@ def test_apply():
 def test_apply_adjoint():
     im_shape = (12, 13)
     num_angles = 10
-    x = jnp.ones(im_shape)
+    x = jnp.ones(im_shape, dtype=jnp.float32)
```

Comment:
This is more of a band-aid than a real fix. The `astra` and `svmbir` linop class initializers explicitly force
```python
    input_dtype=np.float32,
    output_dtype=np.float32,
```
when calling `super().__init__`, but neither of the `scico` ones do. They seem to default to `np.float32`, but it would be better to decide whether fixing it to this choice is desirable, and if so, making it explicit, and properly supporting and testing alternatives such as `np.float64` if it is not.